assets/script.js

```diff
@@ -65,12 +65,17 @@ function getAlbumInfo() {
     console.log(response);
     //album search result shown
     var icon = response.album.image[2]["#text"];
-    var summary = response.album.wiki.summary;
     var albumName = response.album.name;
     $("#album-pic").attr("src", icon);
     $("#summaryHeading").text(albumName);
-    $("#summary").html(summary);
     $(".search-tracks>ol").html("");
+    if (response && response.album['wiki']){
```

Comment:
Here you are evaluating if the response and the summary are === to true which they never will be as they are an object/string. You need to evaluate for false which will include null/undefined

Reply:
`response.album['wiki']` inconsistent notation, why not just use .